artiq/frontend/artiq_dashboard.py

```diff
@@ -209,7 +216,7 @@ def report_disconnect():
         main_window.tabifyDockWidget(d_schedule, d_log0)
 
     logging.info("ARTIQ dashboard %s connected to %s",
-                 artiq_version, args.server)
+                 artiq_version, server_name)
```

Comment:
What happens if the master was not run with ``--name``?

Reply:
`server_name` defaults to `args.server`, so if `--name` is not provided the behaviour is as before.

Reply:
May be better to have both pieces of information in the log.